File: app/src/main/java/com/mdmobile/pocketconsole/gson/Token.java
package com.mdmobile.pocketconsole.gson;


public class Token {

    private String access_token, token_type;
    private int expires_in;


    public Token(String access_token, String token_type, int expires_in) {
        this.access_token = access_token;
        this.token_type = token_type;
        this.expires_in = expires_in;
    }

    public String getAccess_token() {
        return access_token;
    }

    public String getToken_type() {
        return token_type;
    }

    public int getTokenExpiration(){
        return expires_in;
    }
}

File: McApiService/src/main/java/mobicontrol/mcApiService/dataModel/Token.kt
package mobicontrol.mcApiService.dataModel

import com.google.gson.annotations.SerializedName

data class Token(
        @field:SerializedName("access_token")
        var token: String,
        @field:SerializedName("token_type")
        val token_type: String,
        @field:SerializedName("token_expiration")
        val tokenExpiration: Int,
        @field:SerializedName("refresh_token")
        val refreshToken: String)
